src/TimePicker/Clock.js

```diff
@@ -137,11 +138,11 @@ class Clock extends Component {
         width: 260,
         height: 260,
         borderRadius: '100%',
-        backgroundColor: this.context.muiTheme.timePicker.clockCircleColor,
+        backgroundColor: timePicker.clockCircleColor,
```

Comment:
Hmm, we seem to be applying a different amount of fade for the different themes. That could be symptomatic of something else as no other component needs this However given theming is going to get an overhaul, lets leave it for now.